jena-arq/src/main/java/org/apache/jena/riot/rowset/RowSetReaderRegistry.java

```diff
@@ -57,7 +57,8 @@ public static void init() {
         initialized = true;
 
         register(RS_XML,        RowSetReaderXML.factory);
-        register(RS_JSON,       RowSetReaderJSON.factory);
```

Comment:
Leave in-place and commented out. In time, after a release or two, we can drop that but it servers as a quick-fix marker for now.